src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1598,15 +1758,15 @@ private void hitBlockWithToolInHand(@Nullable final BlockPos blockPos, final boo
             if (!worldObj.isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
-                  new BlockParticleEffectMessage(blockPos, worldObj.getBlockState(blockPos), BlockParticleEffectMessage.BREAK_BLOCK),
-                  new NetworkRegistry.TargetPoint(worldObj.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_SOUND_RANGE));
+                        new BlockParticleEffectMessage(blockPos, worldObj.getBlockState(blockPos), BlockParticleEffectMessage.BREAK_BLOCK),
+                        new NetworkRegistry.TargetPoint(worldObj.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_SOUND_RANGE));
             }
             worldObj.playSound(null,
-              blockPos,
-              block.getSoundType(blockState, worldObj, blockPos, this).getBreakSound(),
-              SoundCategory.BLOCKS,
-              block.getSoundType(blockState, worldObj, blockPos, this).getVolume(),
-              block.getSoundType(blockState, worldObj, blockPos, this).getPitch());
+                    blockPos,
+                    block.getSoundType(blockState, worldObj, blockPos, this).getBreakSound(),
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)